src/components/user/profile-activity-graphs.tsx

```diff
@@ -1,44 +1,27 @@
-import { Fragment } from 'react'
+import * as R from 'ramda'
 
 import { ProfileActivityGraph } from '@/components/user/profile-activity-graph'
 import { useInstanceData } from '@/contexts/instance-context'
 import { User } from '@/fetcher/query-types'
 
-interface ProfileActivityGraphsProps {
```

Comment:
Less characters, but annoying if we need to export this later on.
I personally like the verbose version.
 We should decide on a preferred code style together.